### PR TITLE
Storage Patch Release 02/09/2023

### PR DESCRIPTION
**Pull request by ibrahimrabab**

### CODE REVIEW

File: sdk/storage/azure-storage-blob-batch/pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-blob-batch</artifactId>
-  <version>12.17.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-blob-batch;current} -->
+  <version>12.17.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob-batch;current} -->
```
Comment (vcolin7) [inline sdk/storage/azure-storage-blob-batch/pom.xml:16]: We are regressing to an older version here and probably for all "current" tags.

File: sdk/storage/azure-storage-blob-changefeed/pom.xml [hunk 2]
```diff
@@ -84,7 +84,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob</artifactId>
-      <version>12.21.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+      <version>12.21.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
```
Comment (vcolin7) [inline sdk/storage/azure-storage-blob-changefeed/pom.xml:87]: Regressing.

File: sdk/storage/azure-storage-blob-cryptography/pom.xml [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-blob-cryptography</artifactId>
-  <version>12.20.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-blob-cryptography;current} -->
+  <version>12.20.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob-cryptography;current} -->
```
Comment (vcolin7) [inline sdk/storage/azure-storage-blob-cryptography/pom.xml:16]: Regressing.

File: sdk/storage/azure-storage-blob-nio/pom.xml [hunk 4]
```diff
@@ -77,7 +77,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob</artifactId>
-      <version>12.21.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+      <version>12.21.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
```
Comment (vcolin7) [inline sdk/storage/azure-storage-blob-nio/pom.xml:80]: Regressing.

File: sdk/storage/azure-storage-blob/pom.xml [hunk 5]
```diff
@@ -13,7 +13,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-blob</artifactId>
-  <version>12.21.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+  <version>12.21.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
```
Comment (vcolin7) [inline sdk/storage/azure-storage-blob/pom.xml:16]: Regressing.

File: sdk/storage/azure-storage-common/pom.xml [hunk 6]
```diff
@@ -13,7 +13,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-common</artifactId>
-  <version>12.20.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-common;current} -->
+  <version>12.20.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-common;current} -->
```
Comment (vcolin7) [inline sdk/storage/azure-storage-common/pom.xml:16]: Regressing.

File: sdk/storage/azure-storage-file-datalake/pom.xml [hunk 7]
```diff
@@ -13,7 +13,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-file-datalake</artifactId>
-  <version>12.14.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-file-datalake;current} -->
+  <version>12.14.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-file-datalake;current} -->
```
Comment (vcolin7) [inline sdk/storage/azure-storage-file-datalake/pom.xml:16]: Regressing.

File: sdk/storage/azure-storage-file-share/pom.xml [hunk 8]
```diff
@@ -13,7 +13,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-file-share</artifactId>
-  <version>12.17.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-file-share;current} -->
+  <version>12.17.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-file-share;current} -->
```
Comment (vcolin7) [inline sdk/storage/azure-storage-file-share/pom.xml:16]: Regressing.

File: sdk/storage/azure-storage-internal-avro/pom.xml [hunk 9]
```diff
@@ -13,7 +13,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-internal-avro</artifactId>
-  <version>12.6.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-internal-avro;current} -->
+  <version>12.6.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-internal-avro;current} -->
```
Comment (vcolin7) [inline sdk/storage/azure-storage-internal-avro/pom.xml:16]: Regressing.

File: sdk/storage/azure-storage-perf/pom.xml [hunk 10]
```diff
@@ -25,25 +25,25 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob</artifactId>
-      <version>12.21.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+      <version>12.21.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
```
Comment (vcolin7) [inline sdk/storage/azure-storage-perf/pom.xml:28]: Regressing.

File: sdk/storage/azure-storage-queue/pom.xml [hunk 11]
```diff
@@ -13,7 +13,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-queue</artifactId>
-  <version>12.16.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-queue;current} -->
+  <version>12.16.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-queue;current} -->
```
Comment (vcolin7) [inline sdk/storage/azure-storage-queue/pom.xml:16]: Regressing.

**Review by vcolin7 — CHANGES_REQUESTED** [review]
It looks like you are using the `version_client.txt` file from the increment versions PR, instead of updating the one we had on `main` to the versions we just released. You need to update the "dependency" versions to those that end in `.3` and then run the python command to update versions everywhere. "current" versions need to stay the same as they are in `main`, ending in `beta.2`.

File: sdk/storage/azure-storage-blob-changefeed/pom.xml (hunk 2 above)
Comment (ibrahimrabab) [reply sdk/storage/azure-storage-blob-changefeed/pom.xml]: Fixed

**Review by vcolin7 — APPROVED** [review]
:shipit:

**Pull request merged**